core/src/main/java/feast/core/model/Store.java

```diff
@@ -90,6 +81,9 @@ public static Store fromProto(StoreProto.Store storeProto) throws IllegalArgumen
       case REDIS_CLUSTER:
         config = storeProto.getRedisClusterConfig().toByteArray();
         break;
+      case JDBC:
```

Comment:
We probably want to change this to Snowflake so that users dont expect compatibility with all JDBC supported stores.

Reply:
I'm also thinking this branch could be significantly simplified getting rid of all the generic JDBC stuff we started with.  I'm not sure how much value it brings given the implementation we landed on.